content/configuration/tasks-deploy.md

```diff
@@ -185,6 +185,21 @@ specification:
             memory: 500Mi
 ```
 
+## Replicas and Redundancy 
+
+{{< note title="Note" >}}
+For background, a pod comprises a Kubernetes service and container(s). A replica is copy of a pod running on a different node, where a node is a worker machine within a cluster.
```

Comment:
> a pod comprises a Kubernetes service and container(s)

What does that mean?

> A replica is copy of a pod running on a different node, where a node is a worker machine within a cluster.

It's not necessary on another node. I'd say it's "running somewhere else in the cluster".